apps/dav/lib/Migration/CalDAVRemoveEmptyValue.php

```diff
@@ -98,6 +99,37 @@ public function run(IOutput $output) {
 	}
 
 	protected function getInvalidObjects($pattern) {
+		if ($this->db->getDatabasePlatform() instanceof OraclePlatform) {
+			$rows = [];
+			$chunkSize = 500;
+			$query = $this->db->getQueryBuilder();
+			$query->select($query->func()->count('*', 'num_entries'))
+				->from('calendarobjects');
+			$result = $query->execute();
+			$count = $result->fetchColumn();
+			$result->closeCursor();
+
+			$numChunks = ceil($count / $chunkSize);
+
+			$query = $this->db->getQueryBuilder();
+			$query->select(['calendarid', 'uri', 'calendardata'])
+				->from('calendarobjects')
+				->setMaxResults($chunkSize);
+			for ($chunk = 0; $chunk < $numChunks; $chunk++) {
+				$query->setFirstResult($chunk * $chunkSize);
+				$result = $query->execute();
+
+				while ($row = $result->fetchAssociative()) {
+					if (mb_strpos($row['calendardata'], $pattern)) {
```

Comment:
```suggestion
					if (mb_strpos($row['calendardata'], $pattern) !== false) {
```

just to be on the save side